src/polymer/declaration-property-handlers.ts

```diff
@@ -68,6 +68,17 @@ export function declarationPropertyHandlers(
         }
         declaration.observers.push({javascriptNode: element, expression: v});
       }
+    },
+    listeners(node: estree.Node) {
+      if (node.type !== 'ObjectExpression') {
+        return;
+      }
+      for (let prop of node.properties) {
+        if (prop.key.type !== 'Literal' || prop.value.type !== 'Literal') {
```

Comment:
(Optional, for all the bonus points: add Warnings to `document` when any of these constraints don't hold)